Storage/SQL/RemoteTabsTable.swift

```diff
@@ -80,7 +80,7 @@ class RemoteClientsTable<T>: GenericTable<RemoteClient> {
     }
 
     override func getQueryAndArgs(_ options: QueryOptions?) -> (String, Args)? {
-        return ("SELECT * FROM \(name) ORDER BY modified DESC", [])
+        return ("SELECT * FROM \(name) WHERE EXISTS(SELECT 1 FROM \(TableRemoteDevices) rd WHERE rd.guid = fxaDeviceId) ORDER BY modified DESC", [])
```

Comment:
Do we now keep our own FxA device record in the DB? If not, this function now excludes our own client row automatically. Did you check the consequences of that?